app/src/main/java/tk/zedlabs/wallportal/repository/PostDataSource.kt

```diff
@@ -8,68 +8,79 @@ import kotlinx.coroutines.launch
 import tk.zedlabs.wallportal.data.JsonApi
 import tk.zedlabs.wallportal.data.RetrofitService
 import tk.zedlabs.wallportal.models.WallHavenResponse
-import java.lang.Exception
 
 class PostDataSource(private val scope: CoroutineScope) : PageKeyedDataSource<Int, WallHavenResponse>() {
 
-    val PAGE_SIZE = 24
-    val FIRST_PAGE = 1
-    val queryParam = "vaporwave||retrowave||noir||outrun||cyberpunk||japan"
-    val sorting = "views"
-    var  jsonApi : JsonApi = RetrofitService.createService(JsonApi::class.java)
+    companion object {
```

Comment:
what the benefit of `companion object` here?

Reply:
for constants